src/main.cpp

```diff
@@ -3049,13 +3172,14 @@ static CBlockIndex* FindMostWorkChain() {
             if (depth < excessiveAcceptDepth)
 	      {
 		fRecentExcessive |= ((pindexTest->nStatus & BLOCK_EXCESSIVE) != 0);  // Unlimited: deny this candidate chain if there's a recent excessive block
+if ((pindexTest->nStatus & BLOCK_EXCESSIVE) != 0) LogPrintf("recent excessive block\n");
 	      }
             else
 	      {
 		fOldExcessive |= ((pindexTest->nStatus & BLOCK_EXCESSIVE) != 0);  // Unlimited: unless there is an even older excessive block
 	      }
 
-            if (fFailedChain | fMissingData | fRecentExcessive) break;
+            if (fFailedChain | fMissingData | fRecentExcessive) { LogPrintf("breaking ######\n");break;}
```

Comment:
Please format block properly. I assume this is something here for debug and you plan to remove it, feel free to ignore if so.